RussianFilter/sections/specific.txt

```diff
@@ -49,6 +49,8 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+forklog.com##.single_brba
```

Comment:
or `forklog.com##div[style="width: 100%; height: 100%"] > a[target="_blank"] > img`

Reply:
Is `##.single_brba`, enough.